kura/pom.xml

```diff
@@ -130,6 +130,7 @@
         <module>org.eclipse.kura.wire.ai.component.provider</module>
 
         <module>org.eclipse.kura.container.provider</module>
+        <module>org.eclipse.kura.configuration.change.publisher</module>
```

Comment:
I'm not sure if it makes sense to call it configuration change publisher or as generic event publisher and then we will have a snapshot configuration that publishes to the configuration change topic.
Because at its root it is a publisher for the event control topic. Then the app-part is configured to publish to the configuration topic, right?

Would it make sense to rename it as event publisher bundle and have two factories, one for the generic event and one for the configuration change. This last would be pre-filled with the configuration change topic?

Reply:
Agree.